tdrs-backend/None

```diff
@@ -0,0 +1,3 @@
+HEADER20204G06   TAN1 N
+T620204000127470001104500011146000043010000397700003924000084460000706800007222000055042849000055141378000056643253000755810007592100075542000009810000097000000968000392980003934900038972000353020003560200035602001684470016904700168107000464480004649800046203001219990012254900121904000001630000014900000151000003440000033100000276000002580000024100000187000054530000388100003884
+TRAILER0000001         
```

Comment:
Are we using this file? Why is it called None?

Reply:
these files sometimes get generated while running the unit tests, please remove